meetup/services/meetup_api_lookup_agent.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from django.http import HttpResponse
+import requests
+import os
+
+class MeetupApiLookupAgent():
+  def __init__(self, link):
+    self.link = link
+
+  def get_response(self, category=''):
```

Comment:
Is this method meant for public consumption? That is, do you expect other files to import this class and call `get_response` directly? 

I ask because this seems like an internal implementation detail, not something that should be part of the public API of this class. `get_response` doesn't really have to do with the domain of this class, which is information about meetups. If this is not meant to be public, a common Python convention is to prefix the method with an underscore.